internal/release/main.go

```diff
@@ -96,6 +96,7 @@ func generateFile(name, version string) error {
 			*newPlatform(version, "x86_64", "linux", "Debian 9 / Ubuntu 16.04 + 18.04", "deb"),
```

Comment:
unrelated to your change but since are here this is outdated 
```suggestion
			*newPlatform(version, "x86_64", "linux", "Debian 9 / Ubuntu 16.04 + 18.04 + 20.04", "deb"),
```